gateways/cybersource/cybersource.go

```diff
@@ -126,6 +129,26 @@ func buildResponseMetadata(processorInformation ProcessorInformation) map[string
 	return metadata
 }
 
+func buildCreatedTokens(paymentInformation PaymentInformation) map[sleet.TokenType]string {
+	createdTokens := map[sleet.TokenType]string{}
+	if paymentInformation.Customer != nil {
+		createdTokens[sleet.TokenTypeCustomer] = paymentInformation.Customer.ID
+	}
+	if paymentInformation.PaymentInstrument != nil {
+		createdTokens[sleet.TokenTypePayment] = paymentInformation.PaymentInstrument.ID
+	}
+	if paymentInformation.InstrumentIdentifier != nil {
+		createdTokens[sleet.TokenTypePaymentIdentifier] = paymentInformation.InstrumentIdentifier.ID
+	}
+	if paymentInformation.ShippingAddress != nil {
+		createdTokens[sleet.TokenTypeShippingAddress] = paymentInformation.InstrumentIdentifier.ID
```

Comment:
`paymentInformation.ShippingAddress.ID`

Reply:
Already fixed, I just noticed this as I was reading it a second time 😅